test/ui_tests/ViewCapabilityUITest.js

```diff
@@ -0,0 +1,34 @@
+const webdriver = require('selenium-webdriver');
+const chai = require('chai');  
+
+
+
+describe('ViewCapabilityUITest', async () => {
+
+  it('should check if Capability elements on the JobRole page are present', async () => {
+
+    var driver = new webdriver.Builder().
+      withCapabilities(webdriver.Capabilities.chrome()).
+      build();
+
+      await driver.get(process.env.UI_TEST_URL);
+      
+      await driver.findElement(webdriver.By.id('header')).getText().then(function(value) {
+        chai.assert.equal(value, 'View Job Roles')
+      });
+      await driver.findElement(webdriver.By.id('jobs')).getText().then(function(value) {
+        chai.assert.equal(value, 'Jobs')
+      });
+      await driver.findElement(webdriver.By.id('name')).getText().then(function(value) {
+        chai.assert.equal(value, 'Name')
+      });
+      await driver.findElement(webdriver.By.id('specification')).getText().then(function(value) {
+        chai.assert.equal(value, 'Specification')
+      });
+      await driver.findElement(webdriver.By.id('capability')).getText().then(function(value) {
+        chai.assert.equal(value, 'Capability')
+      });
```

Comment:
it looks like this test is identical to the other but with the capability check. can you move this check to the other test run rather than creating a new one?